modules/op-release-notes-1-15.adoc

```diff
@@ -550,3 +550,17 @@ With this update, {pipelines-title} General Availability (GA) 1.15.2 is availabl
 * Before this update, in the web console, if you enabled the {pipelines-shortname} console plugin, when starting a pipeline you could not select the storage class for the volume claim template, because the *StorageClass* list was not present in the *VolumeClaimTemplate* options. With this update, you can select the storage class for the volume claim template.
 
 * Before this update, if you used {pac}, the list of pipeline runs did not display correctly in the *Repository details* page of the web console. With this update, the list of pipeline runs displays correctly.
+
+[id="release-notes-1-15-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.15.3
+
+With this update, {pipelines-title} General Availability (GA) 1.15.3 is available on {OCP} 4.14 and later versions.
```

Comment:
```suggestion
With this update, {pipelines-title} General Availability (GA) 1.15.3 is available on {OCP} 4.12, 4.14 and later versions.
```
We are supporting 1.15.3 from 4.12 as its a EUS but 4.13 we skipped as its Out OF support